code/game/turfs/simulated/floor.dm

```diff
@@ -218,6 +228,14 @@ turf/simulated/floor/update_icon()
 				playsound(src, 'sound/items/bikehorn.ogg', 50, 1)
 				spawn(20)
 					spam_flag = 0
+		//Phazon tiles teleport to another random one in the world when clicked
+		if("phazon")
+			if(!spam_flag)
+				spam_flag = 1
+				var/turf/simulated/floor/destination = pick(phazontiles)
+				do_teleport(user, destination)
+				spawn(20)
```

Comment:
2 SECONDS and setting the flag to TRUE and FALSE would be better but not enough to hold this up over